main/screen.c

```diff
@@ -55,11 +56,17 @@ static lv_obj_t * create_scr_self_test() {
 
     self_test_result_label = lv_label_create(scr);
 
-    self_test_finished_label = lv_label_create(scr);
-    lv_obj_set_width(self_test_finished_label, LV_HOR_RES);
-    lv_obj_add_flag(self_test_finished_label, LV_OBJ_FLAG_HIDDEN);
-    lv_label_set_long_mode(self_test_finished_label, LV_LABEL_LONG_SCROLL_CIRCULAR);
-    lv_label_set_text(self_test_finished_label, "Hold BOOT button for 2 seconds to cancel self test, or press RESET to run again.");
+    self_test_finished_pass_label = lv_label_create(scr);
```

Comment:
Looking it the previous code, it might be simpler to just add the second label here, empty, and set the fail/pass text at line 240/243, instead of having multiple labels.